crates/sui/src/ethereum_client.rs

```diff
@@ -0,0 +1,83 @@
+use clap::Subcommand;
+use serde::{Deserialize, Serialize};
+use sui_types::base_types::ObjectID;
+
+#[derive(Subcommand)]
+pub enum EthClientCommands {
+    /// Approve a TX with Eth contract for a given dWallet.
+    #[command(name = "dwallet-eth-verify")]
+    EthApproveMessage {
+        #[clap(long)]
+        /// Object of a [EthDwalletCap]
```

Comment:
```suggestion
        /// Object of a [EthDwalletCap].
```

All comments start with capital letter and end with a dot.